lapce-data/src/plugin/plugin_install_status.rs

```diff
@@ -0,0 +1,42 @@
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum PluginInstallType {
```

Comment:
This could be `Copy` (with corresponding cleanups), and should use CamelCase naming